### PR TITLE
Newcomers.rst: Add a command pytest to the guide

### PR DESCRIPTION
**Pull request by dorothykiz1**
This command helps to run tests locally
after installing dependencies.

Closes https://github.com/coala/coala/issues/5404

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by palash25** [comment]
@dorothykiz1 you already have a PR open for this issue then why this? https://github.com/coala/coala/pull/5411

We do not like duplicate PRs. You should close one of them.

**Comment by dorothykiz1** [comment]
@palash25 i had no way of editing the other after working on my reviews  , thats why i created this, let me close the former

**Comment by palash25** [comment]
You did have a way of editing the last one. Its a simple command called `git commit --amend`
If you had referred to the docs or ask for help on gitter instead of deciding it for yourself (that it can't be edited) then you would have learnt a new command.

You better learn how to amend your commits because we have to do it for every PR and if you keep making multiple PRs for every little amend a maintainer will have to ban you from contributing till you learn git.

This has happened to a newcomer before.

**Comment by dorothykiz1** [comment]
Thank You

File: docs/Developers/Newcomers_Guide.rst [hunk 1]
```diff
@@ -295,6 +295,20 @@ coala project directory:
 
     $ pip3 install -r test-requirements.txt -r requirements.txt
 
+
+After the test dependencies have been installed
+run pytest to test your changes locally:
+
+::
+
+    $pytest
```
Comment (jayvdb) [inline docs/Developers/Newcomers_Guide.rst]: Add a space after `$` and before `pytest`

**Comment by jayvdb** [comment]
reack https://github.com/coala/coala/pull/5487/commits/ac56dbc687b0e6e3c62d5d201ee6371761b5a5e6

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by dorothykiz1** [comment]
Thank you for merging my pull request looking forward to getting another issue. Am delighted 😊

**Pull request merged**